### PR TITLE
added new mesure for inductances

### PR DESCRIPTION
**Pull request by SasoriOlkof**
Hi there,

I have added a new unit of measure that I need to use :)

Cheers

### CODE REVIEW

**Comment by codingjoe** [comment]
Hi @SasoriOlkof,

interesting, I always love to see people contribute new measures. That being said, isn't capacitance measured in farad (F)? What is the L supposed to stand for?

Best
-Joe

**Comment by SasoriOlkof** [comment]
Hi @codingjoe

the capacity (C) of a capacitor it's merused in farad (F)
the inductace (L) of a inductance (coil) it's mesured in henri (H)
 
:)

**Comment by SasoriOlkof** [comment]
also, I have fixed the error in my file, changing L for H, I had made a mistake of the concept to treat

**Comment by codingjoe** [comment]
Hi @SasoriOlkof thanks for getting back to me so quickly. If I understood you correctly, you want to add `Inductance`. Yet your patch suggests you want to add `Capacitance`, which btw. is already in this package, see https://github.com/coddingtonbear/python-measurement/blob/43b243818e082c295d336dc2c7de23831f7c736e/measurement/measures/capacitance.py#L6-L14

So maybe you could change that.

**Comment by SasoriOlkof** [comment]
Hi @codingjoe, 

the measurement of farad capacity is correct, the measurement of henries for inductances is something else, see here you can see more information :)

https://en.wikipedia.org/wiki/Henry_(unit)

File: measurement/measures/inductance.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from measurement.base import MeasureBase
+
+__all__ = ["Inductance"]
+
+
+class Capacitance(MeasureBase):
```
Comment (codingjoe) [inline measurement/measures/inductance.py:6]:
```suggestion
class Inductance(MeasureBase):
```

File: measurement/measures/inductance.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from measurement.base import MeasureBase
+
+__all__ = ["Inductance"]
```
Comment (codingjoe) [inline measurement/measures/inductance.py:3]: It's funny, here you are referencing the correct class name, even though the class does not exist ;)

**Comment by SasoriOlkof** [comment]
jejejeje, sorry about that

yes it is a good idea to group some measurements, after this I wanted to add the power measurement in watts as well :)

**Comment by codingjoe** [comment]
Hi @SasoriOlkof,

I am working on a complere rewrite of the entire package towards more precise decimals. I already implemented inductance in that version. You can already check it out simply install the latest alpha version `pip install measurement==4.0a3`.

**Comment by SasoriOlkof** [comment]
thanks, I download it now :)

**Pull request closed without merging**